engine/apps/api/serializers/user.py

```diff
@@ -159,7 +166,13 @@ def get_notification_chain_verbal(self, obj: User) -> NotificationChainVerbal:
         return {"default": " - ".join(default), "important": " - ".join(important)}
 
     def get_cloud_connection_status(self, obj: User) -> CloudSyncStatus | None:
-        if settings.IS_OPEN_SOURCE and live_settings.GRAFANA_CLOUD_NOTIFICATIONS_ENABLED:
+        is_open_source_with_cloud_notifications = self.context.get("is_open_source_with_cloud_notifications", None)
```

Comment:
under what circumstances would `context["is_open_source_with_cloud_notifications"]` not be set?

Reply:
I'm not sure 🤔 It will be set for /user and /users endpoints, but I wouldn't completely rely on this, because we reuse this serizalizer so many times, so it's easy to forget to set the context somewhere